modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -110,7 +111,7 @@ if [ -f "$livedev" ]; then
     esac
     [ -e /sys/fs/"$fstype" ] || modprobe "$fstype"
 else
-    livedev_fstype=$(blkid -o value -s TYPE "$livedev")
+    livedev_fstype=$(det_fs "$livedev" "auto")
```

Comment:
Equal to `det_fs "$livedev"`:

https://github.com/dracutdevs/dracut/blob/aca51203609954b38b41ea303a4376cbce061150/modules.d/99fs-lib/fs-lib.sh#L190